webapp/channels/src/components/team_controller/team_controller.tsx

```diff
@@ -100,6 +104,7 @@ function TeamController(props: Props) {
         function handleBlur() {
             window.isActive = false;
             blurTime.current = Date.now();
+            dispatch(unsetActiveChannelOnServer());
```

Comment:
Could you pass this in via `connect`? I don't want to mix the hooks and `connect` in a single component if we can avoid it